tools/scalar-schema/java/src/CosmosTableMetadata.java

```diff
@@ -17,6 +17,7 @@
   private String id;
```

Comment:
Just out of my curiosity. Why do we have CosmosTableMetadata in this schema tool project?

Reply:
The class is used for metadata insertion.
Cosmos DB SDK serializes/deserializes the data according to this class's setter and getter.

This tool gives the instance to the Upsert API.
https://github.com/scalar-labs/scalardb/blob/0f9cd909fd94ac04e46575860d0110ba6d57dab7/tools/scalar-schema/src/scalar_schema/cosmos.clj#L96